Content.Client/Inventory/StrippableBoundUserInterface.cs

```diff
@@ -91,7 +94,13 @@ protected override void UpdateState(BoundUserInterfaceState state)
             Hands = stripState.Hands;
             Handcuffs = stripState.Handcuffs;
 
+
             UpdateMenu();
+
+            if (_strippingMenu != null)
+            {
+                _strippingMenu.SetHeight = (Inventory.Count + Hands.Count + Handcuffs.Count) * 34 + _strippingMenu.MinHeight;
```

Comment:
i'm a bit sus of magic numbers like this 34. Is there not some property of the button that corresponds to this?

Reply:
I'm honestly not sure how best to do this. I've tried getting the height of the _vboxContainer, the height of each of the children of _vboxContainer, the DesiredSize of both, but whenever I do, I get back a height of 0. I assume this is because the button isn't given a Height when it is initialized here:
![image](https://user-images.githubusercontent.com/57637101/183529572-affabd30-0145-4e48-8f09-c42a555dece5.png)
but that said, there might be something I'm missing, as I'm still quite new to the codebase.

Reply:
I've also looked into the debug locals of the _vboxContainer to find:

![image](https://user-images.githubusercontent.com/57637101/183530719-2ce0d562-0b41-46e9-a2ff-61e186251300.png)

which shows that almost nothing about the button is actually set. The height is 0, desiredsize is 0, minheight is 0, maxheight is infinity(!),  etc.

Reply:
Given that `IsArangeValid` and `IsMeasureValid` are both false, the issue is probably that you haven't done an initial measure/arrangement yet, and desired size is only set after you've done that.

But see my other comment: IMO hardcoding the UI size is fine for now.